### PR TITLE
bpo-30076: Rename the BUILD_{TUPLE,MAP}_UNPACK_WITH_CALL opcodes

### PR DESCRIPTION
**Pull request by ZackerySpytz**
https://bugs.python.org/issue30076

### CODE REVIEW

File: Lib/importlib/_bootstrap_external.py [hunk 1]
```diff
@@ -280,7 +282,7 @@ def _write_atomic(path, data, mode=0o666):
 # Whenever MAGIC_NUMBER is changed, the ranges in the magic_values array
 # in PC/launcher.c must also be updated.
 
-MAGIC_NUMBER = (3420).to_bytes(2, 'little') + b'\r\n'
+MAGIC_NUMBER = (3421).to_bytes(2, 'little') + b'\r\n'
```
Comment (serhiy-storchaka) [inline Lib/importlib/_bootstrap_external.py]: No magic number update is needed. The bytecode itself will not be changed, just human names of some opcodes.

File: Lib/test/test_dis.py [hunk 2]
```diff
@@ -441,8 +441,7 @@ def test_boundaries(self):
 
     def test_widths(self):
         for opcode, opname in enumerate(dis.opname):
-            if opname in ('BUILD_MAP_UNPACK_WITH_CALL',
-                          'BUILD_TUPLE_UNPACK_WITH_CALL'):
+            if opname in ('BUILD_VAR_KEYWORD', 'BUILD_VAR_POSITIONAL'):
```
Comment (serhiy-storchaka) [inline Lib/test/test_dis.py]: Remove it at all. This workaround no longer needed.

File: Lib/importlib/_bootstrap_external.py (hunk 1 above)
Comment (ZackerySpytz) [reply Lib/importlib/_bootstrap_external.py]: Hmm, I see.

**Review by serhiy-storchaka — APPROVED** [review]

**Review by ncoghlan — CHANGES_REQUESTED** [review]
The general update here looks good, but I think I made a mistake in my naming suggestion back in 2017 (short version: I now think the new opcodes should be `BUILD_ARG_*` rather than `BUILD_VAR_*`, see the BPO issue for the details).

**Comment by csabella** [comment]
@ZackerySpytz, please address the code review.  Thanks!

**Pull request closed without merging**